src/plotApplications/components/infoCheck/PlotApplicationApplicantInfoCheck.js

```diff
@@ -14,13 +14,21 @@ import {
 import {getFieldOptions, getLabelOfOption} from "../../../util/helpers";
 import {editInfoCheckItem} from "../../actions";
 import {getUserFullName} from "../../../users/helpers";
+import type {Attributes} from "../../../types";
+import {ApplicantTypes} from "../../enums";
+
+type OwnProps = {
+  section: Object,
+  identifier: string,
+  answer: Object
+};
 
 type Props = {
+  ...OwnProps,
```

Comment:
Hmm. This is brilliant practice to separate the ui-related props from the object-related props coming from the BE.

I might need to refactor the CustomDetailedPlan to use same kind of approach. Much more readable component!

Reply:
Indeed. Well, I've mostly separated them just to get the right props exposed for the type coercion of the default export (I believe `connect`/HOC provided props would be marked as missing otherwise), but it does make it a bit more readable too.